circle.yml

```diff
@@ -72,6 +78,12 @@ workflows:
             tags:
               only: /.*/
       - node-8:
+          # Run for all tags (required to allow the deploy to trigger on version tags)
+          filters:
+            tags:
+              only: /.*/
+              only: /.*/
```

Comment:
Do we need to repeat this twice?

Reply:
We don't, I must have grabbed an extra line when copy/pasting